### DISCUSSION TITLE
Reserving package names

### DISCUSSION THREAD

**Discussion by ivodolenc** [npm/feedback#681] [Registry] [answered]
Hi, I've come across a lot of packages, which don't seem to be real projects but just name reservations.

In so-called "projects" there is only the `package.json` file with 3-4 line of code and projects stand empty for months or years with no indication that they will come to life.

Some profiles go to extremes and open over 200-300 projects that are literally 5 different variants of package name.

Is there anything that would prevent this kind of behavior or is it just ok?

**Comment by MylesBorins** [comment, marked as answer]
Our [Open Source Terms](https://docs.npmjs.com/policies/open-source-terms) outline what is appropriate to publish on the registry. We do not actively remove packages that are not malware but the [dispute process](https://docs.npmjs.com/policies/disputes) can be used in some cases to gain access to a name that "you believe is currently misleading or could be confused with a name used by your company or open source project" or "related to your company or open source project that cannot be claimed via account recovery"
